pkg/manager/http.go

```diff
@@ -734,6 +739,48 @@ func (serv *HTTPServer) modulesInfo(w http.ResponseWriter, r *http.Request) {
 	serv.jsonPage(w, r, "modules", cover.Modules)
 }
 
+func (serv *HTTPServer) httpAddInput(w http.ResponseWriter, r *http.Request) {
```

Comment:
How do you upload? Is it possible to do with some command line tool? It would be useful to include a comment with an example of how to upload it.